src/components/CardList/Item.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import Card from '../Card';
+import { INodeItem } from './CardList';
+
+export interface Props {
+  node: INodeItem,
+  selectedId?: number,
+  onToggle?(): void,
+  onSelect?(): void,
+}
+
+export interface State {
+  isToggled: boolean,
+}
+
+class Item extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isToggled: false,
+    };
+  }
+
+  onSelect = () => {
+    alert('Id =' + this.props.node.id + ' ' + 'Name =' + this.props.node.name);
+  }
+
+  onToggle = () => {
+    const isToggled = !this.state.isToggled;
+    this.setState(() => ({
+      isToggled,
+    }));
+  }
+
+  render() {
+
+    let childnodes = null;
+    let style: React.CSSProperties = {};
+    let expandOrCollapse: string;
+
+    const cardActions = [{ content: 'Select', onAction: this.onSelect }];
+    const children = this.props.node.people;
+    const collapseStyle: React.CSSProperties = { fontSize: '20px', color: 'green' };
+    const expandStyle: React.CSSProperties = { fontSize: '25px', color: 'red' };
+
+    if (this.state.isToggled) {
+      style = {
```

Comment:
styles should be in a scss stylesheet and a className can be applied to component if there is a plus or minus. That way we can have a `cursor: pointer` and other styling applied throughout.